Tests/ClientRuntimeTests/ClientRuntimeTests/NetworkingTests/ClientErrorTests.swift

```diff
@@ -101,7 +101,7 @@ class ClientErrorTests: XCTestCase {
     }
 
     func test_waiterErrorType_returnsNilForCRTError() async throws {
-        let crtError = CRTError(code: 2)
+        let crtError = CommonRunTimeError.crtError(.init(code: 2))
```

Comment:
what is `2`?  I know it likely doesn't matter for this test as long as it results in a non-null error, but just want some context for the value used